pkg/storage/parquet/converter_marker.go

```diff
@@ -0,0 +1,60 @@
+package parquet
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"path"
+
+	"github.com/efficientgo/core/errors"
+	"github.com/go-kit/log"
+	"github.com/oklog/ulid"
+	"github.com/thanos-io/objstore"
+	"github.com/thanos-io/thanos/pkg/runutil"
+
+	"github.com/cortexproject/cortex/pkg/storage/tsdb"
+)
+
+const (
+	ConverterMarkerFileName = "parquet-converter-mark.json"
+	CurrentVersion          = 1
```

Comment:
How do we plan to use `CurrentVersion`?
It is a version that bumps when we changes format of the `ConverterMark` file or parquet file

Reply:
I guess we can set the current version of the "parquet file"?

Maybe the parquet common library could expose a const with that info... but for now as it can change quite a bit lets hardcode it here i guess?